### PR TITLE
Document extension

### PR DESCRIPTION
**Pull request by akwanpdf**
Add `documentExtension` when opening doc.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -233,6 +233,21 @@ The file extension for the base64 string in [`document`](#document), if [`isBase
 />
 ```
 
+#### documentExtension
+string, Android only, required if the URL/path does not contain the file extension or if you're loading a Blob/File. Defaults to the extension in the [`document`](#document) prop.
```
Comment (sgong-pdftron) [inline API.md:237]: not android only

Comment (sgong-pdftron) [reply API.md:237]: "if you're loading a Blob/File" unclear what this means? also applies to wrong extension

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md (hunk 1 above)
Comment (akwanpdf) [reply API.md:237]: Oops took that from web. Will fix.

File: API.md [hunk 2]
```diff
@@ -233,6 +233,21 @@ The file extension for the base64 string in [`document`](#document), if [`isBase
...
 ```
 
+#### documentExtension
+string, optional, defaults to the extension in the [`document`](#document) prop. 
+
+Used for specifying the extension of the document to be loaded. 
+
+```js
+<DocumentView
+  document={'pdfInDisguise.png'}
```
Comment (sgong-pdftron) [inline API.md:243]: sample here should be the most common use which is HTTPS link

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md (hunk 2 above)
Comment (sgong-pdftron) [reply API.md:243]: and should use double quote

**Review by JamesTron — APPROVED** [review]

**Review by sgong-pdftron — APPROVED** [review]

**Review by brandenfung2 — APPROVED** [review]

**Pull request merged**